libcaf_net/caf/net/web_socket/framing.cpp

```diff
@@ -116,6 +116,7 @@ class framing_impl : public framing {
   }
 
   void abort(const error& reason) override {
+    ship_closing_message();
```

Comment:
You gave some really good explanation why we sent that here without an explicit code in the PR description. I'd love to see that explanation here as code documentation. 🙂